docs/reference/experiment-config.txt

```diff
@@ -668,7 +683,7 @@ more suitable for large experiments with many trials.
   hyperparameter configuration.
 
 ``max_length``
-  The maximum training length of any one trial, in terms of records, batches or epochs
+  The maximum training length of any one trial, in terms of records, batches, or epochs
```

Comment:
I take it we always use serial commas? Do we have a grammar style guide, ha?

Reply:
I don't think we have a grammar style guide but I personally prefer serial commas.